### ISSUE TITLE
Prettier quick documentation (newlines, bullets, tables?)

### ISSUE THREAD

**Issue by Soothsilver** [Soothsilver/snowride#41]

**Comment by vitorego** [comment]
Yes, we start using SnowRIDE and it is amazing however the keyword documentation is not pretty as compared with RIDE:

![image](https://user-images.githubusercontent.com/28743244/95568258-e9edc280-0a1b-11eb-8d7c-752d9009f73f.png)
